tests/multi_net/ssl_cert_rsa.py

```diff
@@ -0,0 +1,143 @@
+# Simple test creating an SSL connection and transferring some data
+# This test won't run under CPython because it requires key/cert
+
+try:
+    import ubinascii as binascii, usocket as socket, ussl as ssl
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+PORT = 8000
+
+
+# This self-signed key/cert pair is randomly generated and to be used for
+# testing/demonstration only.  You should always generate your own key/cert.
+
+cert = binascii.unhexlify(
```

Comment:
Can you please include as a comment the command use to generate this cert and key.